terraform/environments/delius-iaps/ssm.tf

```diff
@@ -90,3 +90,145 @@ resource "aws_ssm_parameter" "im-interface-soap-odbc-pwd" {
   }
 }
 
+resource "aws_ssm_parameter" "ndelius-interface-user" {
```

Comment:
There's quite a few parameters here.

If they're all related to configuring the same thing, perhaps we can store a single json string (or something structured) with all the values and do a single look up in the user data

Reply:
Yep good idea, or if they need to be separate, maybe just use a for_each in terraform.